sequencers/modeshape-sequencer-ddl/src/main/java/org/modeshape/sequencer/ddl/dialect/teiid/TeiidDdlLexicon.java

```diff
@@ -87,6 +87,103 @@
     }
 
     /**
+     * JCR names related to the teiid materialized options.
+     */
+    public interface MaterializedOptions {
+
+        /**
+         * Whether table is materialized
+         */
+        String MATERIALIZED = "MATERIALIZED";
+
+        /**
+         * Materialized table
+         */
+        String MATERIALIZED_TABLE = "MATERIALIZED_TABLE";
+
+        /**
+         * Whether table is updateable
+         */
+        String UPDATABLE = "UPDATABLE";
+
+        /**
+         * Allow Teiid based management
+         */
+        String ALLOW_MATVIEW_MANAGEMENT = "teiid_rel:ALLOW_MATVIEW_MANAGEMENT";
```

Comment:
Is `teiid_rel` a JCR namespace prefix, or something else? If a ns prefix, then it should use a constant like the other constants in this file.